src/OptionsResolver.php

```diff
@@ -113,7 +113,7 @@ public function getDefault($option)
      */
     public function getDefaults()
     {
-        return $this->defaults;
+        return $this->defaults; // maybe we get to SF to break open the parent?
     }
 
```

Comment:
great to hear :)
good luck over at SF, let me know if I can help out :)

Final thingy, please make this class `final`, even though marked as `@internal` (:+1:) it is still nice to be explicit.